pypit/scripts/run_pypit.py

```diff
@@ -41,6 +41,7 @@ def parser(options=None):
     parser.add_argument("-v", "--verbose", type=int, default=2, help="(2) Level of verbosity (0-2)")
     parser.add_argument("-m", "--use_masters", default=False, action='store_true', help="Load previously generated MasterFrames")
     parser.add_argument("-d", "--develop", default=False, action='store_true', help="Turn develop debugging on")
+    parser.add_argument("--debug_arc", default=False, action='store_true', help="Turn wavelength/arc debugging on")
```

Comment:
I thought about it, but am thinking this will be used rarely and we might
come up with something better for -a.  :)